src/tests/Interop/MarshalAPI/FunctionPointer/FunctionPointer.cs

```diff
@@ -111,4 +112,81 @@ public static void RunGetDelForOutIntTest()
         }
         Assert.Equal(expectedValue, outVar);
     }
+
+    [UnmanagedCallersOnly(CallConvs = [typeof(CallConvCdecl)])]
```

Comment:
Is there a specific need for cdecl?

Reply:
No. I added cdecl here as it's commonly used by the C ABI.

Reply:
I removed the cdecl.

Reply:
It would be simplel to use the default unmanaged calling convention unless there is a reason to override.